scripts/tests/test_preprint_summary.py

```diff
@@ -0,0 +1,37 @@
+import datetime
```

Comment:
1. scripts/tests are not run as part of travis (there purpose is to test that one-off scripts do what is intended). There isn't a great place for celery task tests atm, but Steve's recommendation for sitemap stuff was to add it to a new file in `osf_tests`

2. New test files should use pytest instead of nose